src/main/java/uk/gov/digital/ho/hocs/audit/service/CaseDataExportService.java

```diff
@@ -91,7 +91,7 @@ protected String[] parseData(AuditEvent audit,
     }
 
     @Override
-    @Transactional(readOnly = true)
+    @Transactional
```

Comment:
can it not lock with a readonly transaction?

Reply:
No - limitation of the type of locking.